tests/unittests/DeviceManagerTest.cpp

```diff
@@ -549,13 +551,50 @@ TEST_P(DeviceManagerTest, ReuseModule) {
   EXPECT_TRUE(result2->isEqual(output2));
 }
 
+TEST(DeviceManagerTest, SetDeviceMemory) {
+  // Never set a value.
+  auto configEmpty = DeviceConfig(BackendKind::Interpreter);
+  EXPECT_EQ(configEmpty.getDeviceMemory(), 0);
+  // Has set a value.
+  auto configFull = DeviceConfig(BackendKind::Interpreter);
+  configFull.setDeviceMemory(32768);
+  EXPECT_EQ(configFull.getDeviceMemory(), 32768);
+
+  // Test CPU.
+  auto cpuConfigEmpty = DeviceConfig(BackendKind::CPU);
+  auto cpuConfigFull1 = DeviceConfig(BackendKind::CPU);
```

Comment:
can you make better name? '1' seems unnecessary.

Reply:
> can you make better name? '1' seems unnecessary.

Sure